diff --git a/config.def.h b/config.def.h
index 7c2b10f..4c03952 100644
--- a/config.def.h
+++ b/config.def.h
@@ -1,7 +1,5 @@
 /* modifier 0 means no modifier */
-static char *useragent      = "Mozilla/5.0 (X11; U; Unix; en-US) "
-                              "AppleWebKit/537.15 (KHTML, like Gecko) "
-                              "Chrome/24.0.1295.0 Safari/537.15 Surf/"VERSION;
+static char *useragent      = "Mozilla/5.0 (X11; Linux x86_64; rv:45.0) Gecko/20100101 Firefox/45.0";
 static char *scriptfile     = "~/.surf/script.js";
 static char *styledir       = "~/.surf/styles/";
 static char *cachefolder    = "~/.surf/cache/";
@@ -23,27 +21,20 @@ static Bool strictssl       = FALSE; /* Refuse untrusted SSL connections */
 static time_t sessiontime   = 3600;
 
 /* Webkit default features */
-static Bool enablescrollbars      = TRUE;
+static Bool enablescrollbars      = FALSE;
 static Bool enablespatialbrowsing = TRUE;
 static Bool enablediskcache       = TRUE;
 static int diskcachebytes         = 5 * 1024 * 1024;
 static Bool enableplugins         = TRUE;
-static Bool enablescripts         = TRUE;
+static Bool enablescripts         = FALSE;
 static Bool enableinspector       = TRUE;
-static Bool enablestyle           = TRUE;
+static Bool enablestyle           = FALSE;
 static Bool loadimages            = TRUE;
 static Bool hidebackground        = FALSE;
-static Bool allowgeolocation      = TRUE;
+static Bool allowgeolocation      = FALSE;
+
+#define SETPROP(p) { .v = (char *[]){ "/bin/sh", "-c", "surf-control.sh $0 $1", winid, p, NULL } }
 
-#define SETPROP(p, q) { \
-	.v = (char *[]){ "/bin/sh", "-c", \
-	     "prop=\"`xprop -id $2 $0 " \
-	     "| sed \"s/^$0(STRING) = \\(\\\\\"\\?\\)\\(.*\\)\\1$/\\2/\" " \
-	     "| xargs -0 printf %b | dmenu`\" &&" \
-	     "xprop -id $2 -f $1 8s -set $1 \"$prop\"", \
-	     p, q, winid, NULL \
-	} \
-}
 
 /* DOWNLOAD(URI, referer) */
 #define DOWNLOAD(d, r) { \
@@ -111,10 +102,10 @@ static Key keys[] = {
 	{ 0,                    GDK_Escape, stop,       { 0 } },
 	{ MODKEY,               GDK_o,      source,     { 0 } },
 	{ MODKEY|GDK_SHIFT_MASK,GDK_o,      inspector,  { 0 } },
-
-	{ MODKEY,               GDK_g,      spawn,      SETPROP("_SURF_URI", "_SURF_GO") },
-	{ MODKEY,               GDK_f,      spawn,      SETPROP("_SURF_FIND", "_SURF_FIND") },
-	{ MODKEY,               GDK_slash,  spawn,      SETPROP("_SURF_FIND", "_SURF_FIND") },
+	{ MODKEY,               GDK_g,      spawn,      SETPROP("_SURF_URI") },
+	{ MODKEY|GDK_SHIFT_MASK,GDK_g,      spawn,      SETPROP("_SURF_URI_RAW") },
+	{ MODKEY,               GDK_f,      spawn,      SETPROP("_SURF_FIND") },
+	{ MODKEY,               GDK_b,      spawn,      SETPROP("_SURF_BMARK") },
 
 	{ MODKEY,               GDK_n,      find,       { .b = TRUE } },
 	{ MODKEY|GDK_SHIFT_MASK,GDK_n,      find,       { .b = FALSE } },
@@ -126,7 +117,6 @@ static Key keys[] = {
 	{ MODKEY|GDK_SHIFT_MASK,GDK_a,      togglecookiepolicy, { 0 } },
 	{ MODKEY|GDK_SHIFT_MASK,GDK_m,      togglestyle, { 0 } },
 	{ MODKEY|GDK_SHIFT_MASK,GDK_b,      togglescrollbars, { 0 } },
-	{ MODKEY|GDK_SHIFT_MASK,GDK_g,      togglegeolocation, { 0 } },
 };
 
 /* button definitions */
